src/main.zig

```diff
@@ -48,9 +48,18 @@ fn loop(
     var buffered_writer = std.io.bufferedWriter(std_out);
     const writer = buffered_writer.writer();
 
+    var arena = std.heap.ArenaAllocator.init(std.heap.page_allocator);
```

Comment:
switched from `server.allocator` to `std.heap.page_allocator` here

Reply:
Thats intentional. Is that a problem?

Reply:
Why is it intentional - would it be incorrect to use the server GPA? (Maybe like a metadata or perf thing?) :)

Reply:
Your change sounds good to me! :)